bigframes/core/indexes/base.py

```diff
@@ -134,7 +134,6 @@ def name(self, value: blocks.Label):
 
     @property
     def names(self) -> typing.Sequence[blocks.Label]:
-        """Returns the names of the Index."""
```

Comment:
For context for Huan, these are borrowed from pandas, so need to come from `third_party`.